templates/publisher/listing.html

```diff
@@ -303,27 +303,22 @@
               <div class="col-12">
                 <div class="p-notification--negative">
                   <p class="p-notification__response">
-                      <strong>Screenshots:</strong> {{ field_errors['screenshots'] }}
+                      <strong>Media:</strong> {{ field_errors['screenshots'] }}
                   </p>
                 </div>
               </div>
             </div>
           {% endif %}
           <div class="row" data-validation-name="screenshots">
-            <div class="col-12">
-              <div class="p-screenshots-toolbar" id="screenshots-toolbar">
-                <label>Screenshots (up to 5):</label>
-                <div class="p-screenshots-toolbar__buttons">
-                  <button class="p-button-neutral js-add-screenshots"><i class="p-icon--plus"></i></button>
-                  <button class="p-button-neutral js-delete-screenshot"><i class="p-icon--delete"></i></button>
-                  <button class="p-button-neutral js-fullscreen-screenshot  u-no-margin--top"><i class="p-icon--fullscreen"></i></button>
-                </div>
+            <div class="col-2">
+              <label>Images:</label>
```

Comment:
Here it says 'Images', but couple lines above it says 'Media' (in error notification) - is this intended?

Reply:
I think it should be "Images" in the notification for now. Will update :+1: